scipy/meson.build

```diff
@@ -111,9 +111,16 @@ numpy_nodepr_api = '-DNPY_NO_DEPRECATED_API=NPY_1_9_API_VERSION'
 blas = dependency('openblas')
 lapack = dependency('openblas')
 
-# TODO: deal with case where we do need the g77 ABI wrappers (see
-#       _build_utils/_fortran.py)
-g77_abi_wrappers = files('_build_utils' / 'src' / 'wrap_dummy_g77_abi.f')
+libname = get_option('libname')
+
+if libname == 'mkl'
```

Comment:
This should then use the found dependency (from lines 111/112), like `blas.name` (see https://mesonbuild.com/Reference-manual_returned_dep.html#depname)